gfsphysics/physics/GFS_debug.F90

```diff
@@ -544,6 +546,30 @@ subroutine print_int_0d(mpirank,omprank,blkno,name,var)
 
       end subroutine print_int_0d
 
+      subroutine print_logic_1d(mpirank,omprank,blkno,name,var)
+
+          use machine,               only: kind_phys
+
+          implicit none
+
+          integer, intent(in) :: mpirank, omprank, blkno
+          character(len=*), intent(in) :: name
+          logical, intent(in) :: var(:)
+
+          integer :: i
+
+#ifdef PRINT_SUM
+          write(0,'(2a,3i6,2i8)') 'XXX: ', trim(name), mpirank, omprank, blkno, size(var), count(var)
+#elif defined(PRINT_CHKSUM)
+          write(0,'(2a,3i6,2i8)') 'XXX: ', trim(name), mpirank, omprank, blkno, size(var), count(var)
```

Comment:
What's the difference between lines 562 and 564?

Reply:
There is no difference for logical variables, strictly speaking we could combine those two. For real and integers, however, it does something different.